src/main/java/com/emc/pravega/perf/PravegaPerfTest.java

```diff
@@ -173,24 +166,44 @@ public static void main(String[] args) {
                                                        .filter(x -> x != null)
                                                        .flatMap(x -> x.stream())
                                                        .collect(Collectors.toList());
-            fjexecutor.invokeAll(workers);
-            fjexecutor.shutdown();
-            fjexecutor.awaitTermination(runtimeSec, TimeUnit.SECONDS);
-            endTime = Instant.now();
-            if (produceStats != null) {
-                produceStats.printTotal(endTime);
-            }
 
-            if (consumeStats != null) {
-                consumeStats.printTotal(endTime);
-            }
+            Runtime.getRuntime().addShutdownHook(new Thread() {
+                public void run() {
+                    try {
+                        System.out.println();
+                        shutdown();
+                    } catch (InterruptedException e) {
+                        e.printStackTrace();
+                    }
+                }
+            });
+
+            fjexecutor.invokeAll(workers);
+            shutdown();
         } catch (Exception e) {
             e.printStackTrace();
         }
 
         System.exit(0);
     }
 
+    private static synchronized void shutdown() throws InterruptedException {
+        final Instant endTime=Instant.now();
```

Comment:
Minor: Add an space before and after `=`. Maybe we need to import the checkstyle from Pravega repository to this one as well.